metadata-ingestion/tests/integration/business-glossary/glossary_events_golden.json

```diff
@@ -145,6 +145,20 @@
         "runId": "datahub-business-glossary-2020_04_14-07_00_00"
     }
 },
+{
+    "entityType": "glossaryTerm",
+    "entityUrn": "urn:li:glossaryTerm:Classification.Highly Confidential",
+    "changeType": "UPSERT",
+    "aspectName": "domains",
+    "aspect": {
+        "value": "{\"domains\": [\"urn:li:domain:en.wikipedia.org\"]}",
```

Comment:
better example would be `urn:li:domain:Marketing`

Reply:
done